joomla/joomla-cms
# [5.0] actionlogs for custom fields

### Summary of Changes
Logs fields in the user action logs


### Testing Instructions
clean install of this branch or apply pr and run update sql
create a field
edit a field
check the two actions are in the user action log

### Actual result BEFORE applying this Pull Request
nothing is displayed in the users action log component


### Expected result AFTER applying this Pull Request
Entries displayed with clickable links

_if nothing is displayed then open the actionlogs options and check to see if fields is in the list of loggable extensions and if not then add it._


### Documentation Changes Required
none
